exodus_lambda/functions/base.py

```diff
@@ -18,22 +18,12 @@ def conf(self):
             if isinstance(self._conf_file, dict):
                 self._conf = self._conf_file
             else:
-                for conf_file in [
-                    self._conf_file,
-                    os.path.join(
-                        os.path.dirname(
-                            os.path.dirname(os.path.dirname(__file__))
-                        ),
-                        "configuration",
-                        "lambda_config.json",
-                    ),
-                ]:
-                    if os.path.exists(conf_file):
-                        with open(
-                            conf_file, "r", encoding="UTF-8"
-                        ) as json_file:
-                            self._conf = json.load(json_file)
-                        break
+                # to be polished: what if the self._conf_file not exist
+                if os.path.exists(self._conf_file):
```

Comment:
Hmm. Maybe it does make sense to remove that default, since the config in that file isn't usable as-is.
In the case self._conf_file doesn't exist, maybe it's best to just exit/fail rather than assume any details about the project.

Reply:
IMO it makes sense to remove it but I thought we were going to rename the file as well to make it even clearer.

The reality is that the lambda_config.json checked into the repo is NOT a config file, it's a template used to generate a config file.  I think the JIRA mentioned renaming the file at one point to make it clear, but did the idea get lost?  It still makes sense to me and then it'd be absolutely obvious that trying to directly load the file as config is wrong.